pyproject.toml

```
@@ -1,3 +1,10 @@
 [build-system]
 requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
+
+[tool.autopep8]
+max_line_length = 120
+# ignore = "E501,W6"
```

What about setting here the errors and warnings to ignore?
```suggestion
ignore = [
	"E221",  # multiple spaces before operator
	"E241",  # multiple spaces after ':'
	"E704",  # multiple statements on one line (def)
	"W503"   # line break before binary operator
]
```